src/DependencyInjection/CmfSeoExtension.php

```diff
@@ -52,7 +52,7 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
```

Comment:
hm, did not even change that actively. i wonder if phpstorm did that on save or what :-|